### ISSUE TITLE
dev-haskell/config-value upgrade conflict

### ISSUE THREAD

**Issue by trofi** [gentoo-haskell/gentoo-haskell#1105]
World upgrade fails due to `config-value` conflicts:

```
dev-haskell/config-value:0

  (dev-haskell/config-value-0.8.1:0/0.8.1::haskell, ebuild scheduled for merge) USE="doc hscolour profile test -hoogle" ABI_X86="(64)" pulled in by
    >=dev-haskell/config-value-0.8:=[profile?] required by (dev-haskell/config-schema-1.2.1.0:0/1.2.1.0::haskell, ebuild scheduled for merge) USE="doc hscolour profile test -hoogle" ABI_X86="(64)"
    ^^                         ^^^                                                                                                                                                              

  (dev-haskell/config-value-0.7.0.1:0/0.7.0.1::haskell, ebuild scheduled for merge) USE="doc hscolour profile test -hoogle" ABI_X86="(64)" pulled in by
    <dev-haskell/config-value-0.8:=[profile?] required by (dev-haskell/glirc-2.35:0/2.35::haskell, ebuild scheduled for merge) USE="doc hscolour profile test -hoogle" ABI_X86="(64)"
    ^     
```

**Comment by ezzieyguywuf** [comment]
@trofi i can look into this and resve the conflicts

**Comment by trofi** [comment]
Sounds good!

**Comment by ezzieyguywuf** [comment]
I don't think I have the appropriate permissions to close issues, but this issue should be closed now with #1108

**Comment by solpeth** [comment]
Closing.
